app/controllers/users/verify_account_controller.rb

```diff
@@ -42,5 +43,10 @@ def decrypted_pii
         cacher.fetch
       end
     end
+
+    def next_step
+      return after_sign_in_path_for(current_user) if user_session['openid_auth_request'].present?
+      sign_up_completed_url
```

Comment:
My understanding is that we are getting rid of this screen, so might as well do it here, at least for this scenario. By "getting rid of it", I mean always redirect to sign_up_completed_url and don't include any logic for OIDC.